src/extensions/averagewaveforms/AverageWaveformsView/AverageWaveformsToolbar.tsx

```diff
@@ -0,0 +1,51 @@
+import { IconButton } from '@material-ui/core';
```

Comment:
Also largely cribbed from `TimeWidgetToolbarNew.tsx`. This can probably be generalized to just be a shared implementation of a toolbar.

Reply:
Yes, it doesn't really have anything to do with average waveforms. So we can pull this out to something like ViewToolbar in common... and pass in the custom actions.